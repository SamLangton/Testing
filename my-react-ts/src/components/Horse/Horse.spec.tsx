/*
	Snapshot tests for "./Horse.tsx"
*/
import * as React from 'react';
import Horse from './Horse';
import * as renderer from 'react-test-renderer';
import { getRacerData } from '../../utils';

describe('./Horse.tsx', () => {

	it('should render as expected', () => {
		let props = {
			Racer: {getRacerData}

		}
		const component = renderer.create(
			<Horse racer={props[0]}/>
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});