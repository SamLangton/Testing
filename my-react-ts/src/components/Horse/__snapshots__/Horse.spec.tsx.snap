// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./Horse.tsx should render as expected 1`] = `
<div
  className="Horse"
  style={
    Object {
      "paddingLeft": "0px",
    }
  }
>
  <img
    height="100px"
    src={undefined}
  />
</div>
`;
